dcpy/lifecycle/package/generate_metadata_assets.py

```diff
@@ -0,0 +1,31 @@
+from dcpy.models.product.dataset.metadata import Metadata, SocrataMetada
```

Comment:
Hakuna metada (@alexrichey I assume the Metada class will be retired or renamed by your PR?)

Reply:
Ha, yes, something like that. It'll probably just be moved to a DCP-specific Socrata model class. Sort of an extension of the base metadata